src/main/kotlin/no/nav/syfo/client/pdfgen/model/PdfModelEndringTidStedArbeidstaker.kt

```diff
@@ -10,4 +10,5 @@ data class EndringTidStedArbeidstaker(
 
 data class EndringTidStedArbeidstakerTidOgSted(
     val sted: String,
+    val videoLink: String
```

Comment:
Disse videoLink og fritekst-feltene kan muligens defineres med type `String?` siden de ikke er påkrevd.

Reply:
Tenker du her (relatert til grensesnittet mot pdf-genereringen) eller over hele linja?

Reply:
Jeg tenker vel egentlig over hele linja, men primært på NewDialogmoteDTO (og sub-DTOer) som kommer fra klienten og evt ned i databasen. Har ikke tenkt noe på pdf-genereringen, men det må vel tas høyde for at disse verdiene ikke er påkrevd der også vil jeg tro.

Reply:
OK, kan definere de nye feltene som optional med ? over hele linja i dto'ene. Men vil gjerne beholde NOT NULL DEFAULT '' i db-skjemaet, og i så fall vil det leses ut som tomme strenger (fra get-tjenesten) dersom de ikke har fått noen annen verdi. Høres det ok ut?

Reply:
OK det, men er det noen grunn til å ikke ha de nullable i db også? Ingen big deal for så vidt, bare lurer :)

Reply:
Blir erfaringsmessig minst problemer av å ha NOT NULL i databasen, med mindre man har en god grunn til noe annet. En god grunn kunne feks være at man i klienten vil at noe skal oppføre seg forskjellig avhengig av om verdien er NULL eller tom streng, men jeg vet ikke om det er aktuelt for disse tre nye feltene.